### PR TITLE
Added changelogs as part of GitHub release workflow.

### PR DESCRIPTION
**Pull request by rohitsngh27**
Created changelog using a metcalfc/changelog-generator and then adding it to a file. Also including it as part of github release which will be created when the workflow runs.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -44,3 +53,4 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           artifacts: /home/runner/work/blueflood/blueflood/blueflood-all/target/blueflood-all-${{ github.event.inputs.version }}-jar-with-dependencies.jar
           tag: "blueflood-${{ github.event.inputs.version }}"
+          body: "${{ steps.changelog.outputs.changelog }}"
```
Comment (zzantozz) [inline .github/workflows/release.yml:56]: Something here doesn't look right. Either the changelog file will always have only the latest changes in it every time, or every release will have the entire history of the project in it. The changelog file and release body should be treated differently.

Comment (rohitsngh27) [reply .github/workflows/release.yml:56]: Changelog file does have the latest changes in it. What do we want in release body?

Comment (zzantozz) [reply .github/workflows/release.yml:56]: Only the changes for that release, right?

Comment (rohitsngh27) [reply .github/workflows/release.yml:56]: which means all the commits since last release?

Comment (zzantozz) [reply .github/workflows/release.yml:56]: Well, with the way we're going now, yes. What's important is that a release notifies users what's new in the release.

Comment (rohitsngh27) [reply .github/workflows/release.yml:56]: So I tested it on my fork and release body does have the commits list since last release.

Comment (zzantozz) [reply .github/workflows/release.yml:56]: So do you want to go ahead and merge this like this and then figure out where to go from there?

**Review by zzantozz — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (rohitsngh27) [reply .github/workflows/release.yml:56]: Sure... I will figure out a way.

**Pull request merged**